client/router_create.go

```diff
@@ -567,7 +567,7 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 		AuthenticatePeer: true,
 	})
 
-	routerConfig.AddSimpleSslProfileWithPath("/etc/skupper-router-certs",
+	routerConfig.AddSimpleSslProfileWithPath("/etc/skupper-shared-certs",
```

Comment:
I thought that if the config-sync can not access to the skupper internal certs (because the shared volume is mapped to a different path), would be safer in the sense of making sure that the config-sync can only access to the certs it manages.

On the other hand, I understand that having all the certs that the router needs in the same place is more clear.

If you do not see any issue on sharing the path between volume secrets and the new empty volume, I can change it.

Reply:
Done!